Client/Coordinators/Scene/SceneCoordinator.swift

```diff
@@ -37,6 +37,16 @@ class SceneCoordinator: BaseCoordinator, LaunchFinishedLoadingDelegate {
         startLaunch(with: launchType)
     }
 
+    /// Handles the specified route.
+    ///
+    /// - Parameter route: The route to handle.
+    ///
+    func handle(route: Route) {
+        // TODO: Implement this function.
+    }
+
+    // MARK: - OpenURLDelegate
```

Comment:
Code has changed and the ` // MARK: - OpenURLDelegate` doesn't apply anymore. I would also put this `handle(route:` function under another MARK, below `launchBrowser` function
```suggestion
    /// Handles the specified route.
    ///
    /// - Parameter route: The route to handle.
    ///
    func handle(route: Route) {
        // TODO: Implement this function.
    }
```